packages/editor/src/components/editor-help/index.native.js

```diff
@@ -68,6 +68,7 @@ function EditorHelpTopics( { close, isVisible, onClose } ) {
 			hideHeader
 			hasNavigation
 			contentStyle={ styles.contentContainer }
+			testID="editor-help-modal"
```

Comment:
Test ID added as the bottom sheet modal does not provide any user-facing identifier from which to query this element. This test ID is used to trigger actions like dismissing the modal via swipe, overlay press, or hardware back button.